### DISCUSSION TITLE
with peewee ORM, the normal function can depends get_db?

### DISCUSSION THREAD

**Discussion by zhangwk02** [fastapi/fastapi#6960] [Questions] [answered]

**Comment by Mause** [comment, marked as answer]
Please follow the issue template so we can help you

**Comment by tiangolo** [comment]
Sorry, I (we) can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.
